include/LISTS_AND_COLLECTIONS.md

```diff
@@ -0,0 +1,9 @@
+# Lists and Collections
```

Comment:
```suggestion
# Lists
```

Reply:
It's intended that the Lists section be for actual lists (e.g. awesome lists) not only from a data structure's point of view